src/components/Avatar/Avatar.component.tsx

```diff
@@ -39,6 +40,24 @@ export const CAMERA = {
     }
   }
 };
+export interface SpawnState {
+  onMountEffect?: {
+    src: string | null;
+    animationSrc?: string | null;
+    loop?: number | null;
+  } | null;
+  onMountAnimation?: {
+    src: string | null;
+    loop?: number | null;
+  } | null;
+}
+
+const initialSpawnState: SpawnState = {
+  onMountEffect: null,
+  onMountAnimation: null
+};
```

Comment:
Maybe this would be better off as a generic store which we would use in any component.

Reply:
`src/state/spawnAtom.ts`